README.md

```diff
@@ -3,6 +3,7 @@ Metal³ Development Environment
 
 This repository includes scripts to set up a Metal³ development environment.
 
-# Instructions
```

Comment:
why change this ? And please do this in a separate commit

Reply:
Because of lint-md fail. It requires to have one common way of highlighting